File: Programming-Languages/Python/Advanced/regular expressions.py
import re

text = "My email is example@mail.com"
pattern = r"\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,}\b"

match = re.search(pattern, text)
if match:
    print("Email Found:", match.group())  # Output: example@mail.com

File: Programming-Languages/C-C++/Advanced/regular expressions.cpp
#include <iostream>
#include <regex>
using namespace std;

int main()
{
    string text = "Email: test@example.com";
    regex pattern("(\\w+@[a-zA-Z_]+?\\.[a-zA-Z]{2,6})");

    if (regex_search(text, pattern))
        cout << "Valid Email Found!" << endl;
    else
        cout << "No Valid Email!" << endl;

    return 0;
}
